app/views/school_groups/digital_signage/index.html.erb

```diff
@@ -0,0 +1,60 @@
+<%= content_for :page_title, t('pupils.digital_signage.index.title') %>
+
+<div class="row">
+  <div class="col-12">
+    <h1><%= t('pupils.digital_signage.index.title') %></h1>
+  </div>
+</div>
+
+<div class="row">
+  <div class="col-12">
+    <%= render PromptComponent.new status: :neutral, icon: 'info-circle' do |c| %>
+      <% c.with_link { link_to 'Get in touch', 'mailto:hello@energysparks.uk' } %>
```

Comment:
Do these bits of text need translating?

Reply:
Yes, that's me rushing! :man_facepalming: